CMakeLists.txt

```diff
@@ -190,6 +193,10 @@ else()
 	endmacro(enable_pch)
 endif()
 
+if(NOT ENABLE_INNOEXTRACT)
+	add_definitions(-DVCMI_NO_INNOEXTRACT)
```

Comment:
Why not calling it VCMI_INNOEXTRACT? (Without negation)

Reply:
Editor and launcher use -DENABLE_EDITOR / -DENABLE_EDITOR definition. So by same logic this one should be named -DENABLE_INNOEXTRACT

Reply:
Yes, that's the reason I preferred this.

Reply:
Sorry, I don't get the reason :) 

Also, the define should be added only to the launcher target.

Reply:
Ah, sorry. Misunderstood it.

The reason for this way was `VCMI_NO_EXTRA_VERSION`. Used this as reference.